static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md

```diff
@@ -33,17 +33,17 @@ permissions.
 This step is optional. You can skip it if you are setting up a new DVC
 repository and don't have your local cache stored in `.dvc/cache`. If you did
 work on your project with DVC previously and you wish to transfer your cache to
-the shared cache directory (external to your <abbr>workspace</abbr>), you will
-need to simply move it from an old cache location to the new one:
+the shared cache directory (external to your workspace), you will need to simply
+move it from an old cache location to the new one:
 
 ```dvc
 $ mv .dvc/cache/* /path/to/dvc-cache
 ```
 
-### Configure External Cache
+### Configure External Cache (Optional)
```

Comment:
I've removed it optionality of this. It's not an optional step for this guide.

Reply:
it's better to use `Configure Shared Cache`

Reply:
Fixed in 0307e74